homeassistant/components/bluetooth/manager.py

```diff
@@ -18,7 +18,7 @@
 )
 
 from homeassistant import config_entries
-from homeassistant.components.logger import EVENT_LOGGING_CHANGED
+from homeassistant.const import EVENT_LOGGING_CHANGED
```

Comment:
Does the bluetooth integration still need the logger integration as dependency?

Reply:
A test failed when I removed it

Reply:
OK I took a look, I think it is safe to update the test and remove the dependency 👍🏾